staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/aes/aes.go

```diff
@@ -48,6 +52,64 @@ func NewGCMTransformer(block cipher.Block) value.Transformer {
 	return &gcm{block: block}
 }
 
+// NewGCMTransformerFromConfig takes a configuration and creates a prefix transformer to
+// handle AEAD encryption for multiple keys
+// Example configuration:
+// keys:
+//   - name: key1
+//     file: key1.txt
```

Comment:
1. Some users may want keys which are represented in bytes for security. It may be awkward to have byte keys in a configuration file. Alternatively, we can define a convention to paste keys in base64 here.

2. This way, keys can be kept away from the configuration. Configuration file may be accessible to multiple users / groups, but the keys should only be readable by the APIserver user preferably. The file based approach allows a certain extent of privilege separation.